.github/workflows/bazel.yml

```diff
@@ -291,9 +291,37 @@ jobs:
           run: |
             cd /workspaces/magma
             bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}"
-            bazel build lte/gateway/release:sctpd_deb_pkg lte/gateway/release:magma_deb_pkg \
+            bazel run //lte/gateway/release:release_build \
```

Comment:
We might let `release_build` take the output path as parameter, so we don't have to know that it's `/tmp/packages`

Reply:
or just put them inside the working directory

Reply:
discussed offline - will stay as is for now